generator/beat/{beat}/Makefile

```diff
@@ -2,47 +2,78 @@ BEAT_NAME={beat}
 BEAT_PATH={beat_path}
 BEAT_GOPATH=$(firstword $(subst :, ,${GOPATH}))
 BEAT_URL=https://${BEAT_PATH}
-SYSTEM_TESTS=false
-TEST_ENVIRONMENT=false
-ES_BEATS?=./vendor/github.com/elastic/beats
-GOPACKAGES=$(shell govendor list -no-status +local)
-PREFIX?=.
-NOTICE_FILE=NOTICE
-
-# Path to the libbeat Makefile
--include $(ES_BEATS)/libbeat/scripts/Makefile
+BEAT_DOC_URL=https://path/to/{beat}/doc
+BEAT_DESCRIPTION=My description
+BEAT_LICENSE=My license
+COPYRIGHT_TXT=My company
+COPYRIGHT_YEAR={current_year}
+VERSION=0.1.0
```

Comment:
Should this be by default the same version as Beats?

Reply:
Up-to-you to decide. But my own preference is  to have a different version number between the application and the libbeat library.